src/charts/utilization-bar/utilization-bar-chart.html

```diff
@@ -2,14 +2,14 @@
   <span ng-if="!layout || layout.type === 'regular'">
     <div ng-if="chartTitle" class="progress-description">{{chartTitle}}</div>
     <div class="progress progress-label-top-right" ng-if="chartData.dataAvailable !== false">
-      <div class="progress-bar" role="progressbar" ng-class="{'animate': animate,
+      <div class="progress-bar" aria-valuenow="{{chartData.percentageUsed}}" aria-valuemin="0" aria-valuemax="100" ng-class="{'animate': animate,
            'progress-bar-success': isOk, 'progress-bar-danger': isError, 'progress-bar-warning': isWarn}"
            ng-style="{width:chartData.percentageUsed + '%'}" tooltip="{{chartData.percentageUsed}}% Used" >
         <span ng-if="chartFooter" ng-bind-html="chartFooter"></span>
         <span ng-if="!chartFooter && (!footerLabelFormat || footerLabelFormat === 'actual')"><strong>{{chartData.used}} of {{chartData.total}} {{units}}</strong> Used</span>
         <span ng-if="!chartFooter && footerLabelFormat === 'percent'"><strong>{{chartData.percentageUsed}}%</strong> Used</span>
       </div>
-      <div class="progress-bar progress-bar-remaining" role="progressbar" aria-valuenow="5" aria-valuemin="0" aria-valuemax="100"
+      <div class="progress-bar progress-bar-remaining"
```

Comment:
Well, I've done that in 2d0f11e, but then dropped the aria-* values in 638b2d9. Reason is explained in the commit messages.

I did it explicitly in multiple commits to show how my reasoning evolved. But also to be able to drop the the last commit (638b2d9) if you don't agree with my reasoning.